config/locales/en/breadcrumbs.yml

```diff
@@ -6,13 +6,13 @@ en:
         new: :'course.levels.new.header'
       admin:
         controller:
-          admin: 'Administration'
+          admin: :'system.admin.admin.index.header'
```

Comment:
Shouldn't this reference the course admin header, and that reference the global admin header instead?